packages/corvid-local-site/src/sitePaths.js

```diff
@@ -69,10 +86,16 @@ const isPathOfPageStructure = (localFilePath, pageId = null) => {
 
 const isPathOfPageFile = (localFilePath, pageId = null) =>
   isPathOfPageCode(localFilePath, pageId) ||
-  isPathOfPageStructure(localFilePath, pageId);
+  isPathOfPageStructure(localFilePath, pageId) ||
```

Comment:
@nemni8 	@dor-itzhaki 	we should rename "isPathOfPageFile" to be something like "isPathOfPaggeRealtedFile"